src/main/kotlin/pt/up/fe/ni/website/backend/model/Project.kt

```diff
@@ -0,0 +1,21 @@
+package pt.up.fe.ni.website.backend.model
+
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import javax.persistence.Entity
+import javax.persistence.EntityListeners
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+
+@Entity
+@EntityListeners(AuditingEntityListener::class)
+class Project(
+    var name: String,
+    var description: String,
+    @Id @GeneratedValue
+    val id: Long? = null
+) {
+    companion object {
+        @JvmStatic
+        val BLANK_PROJECT = Project("", "")
+    }
```

Comment:
This is the only part I don't really like about this implementation. I think a `create` function in `Dto` would be a better abstraction, it seems weird to me to have this in the model and having to use it in the controller.

If the problem is you don't have a `createValue` the same way you have an `updateValue` (I didn't find it but I didn't look too much into it) then I think the `ModelMapper` could be a better option. If you decide to not use it then here's a solution from the top of my head:
- Create an abstract function in `Dto` called `blankEntity()`
- Create a wrapper `fun create() = update(blankEntity())`
- Override `blankEntity()` in `PostDto` instead of adding it to the model